formulas/github-get-archive/README.md

```diff
@@ -0,0 +1,10 @@
+github-get-archive
+==================
+
+This formula adds a function that allows you to pull down a Github, Inc. repository in tar format.
+
+For more information on this function see [github-get-archive](functions/github-get-archive).
+
+Returns nothing.
+
+
```

Comment:
The function's description should have been included automatically.  There's a problem.  Is the function executable?